_posts/2022/2022-10-03-date-and-time-extensions-are-an-unsung-hero-of-active-support.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Time extensions are an unsung hero of Active Support"
+description: "There’s a method for that"
+layout: article
+category: ruby
+image:
+  base: "2022/time-ranges-are-an-unsung-hero-of-active-support"
+  alt: "Calendar with pins"
+  credit: "Towfiqu barbhuiya"
+  source: "https://unsplash.com/photos/bwOAixLG0uc"
+
+---
+
+In Active Support‘s extensions to the core Ruby classes, some of the most useful and plentiful are related to `Time`.
+
+One of Rails’ founding uses was to provide helpful methods to provide reusable methods for regularly performed tasks. In our web applications we‘re often using time periods.
+
+Enter the methods from [DateAndTime::Calculations](https://github.com/rails/rails/blob/main/activesupport/lib/active_support/core_ext/date_and_time/calculations.rb), or see the [API documentation](http://api.rubyonrails.org/classes/DateAndTime/Calculations.html) or the [`DateTime` section in the Core Extensions Rails guide](http://guides.rubyonrails.org/active_support_core_extensions.html#extensions-to-datetime).
```

Comment:
Not sure about the "or see" structure here. Maybe: "Enter the methods....! And then separately something along the lines of: to find out more, check out the API docs.. but maybe that "Api docs" link should come later down. Haven't finished reading yet.